spec/services/splitwise_service_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe SplitwiseService do
+  def build(*_args)
+    described_class.new(*_args)
+  end
+
+  pending "describe what your service does here"
```

Comment:
No tengo claro como están manejando el tema de los tests, pero recomiendo que esto quede testeado.

Reply:
Por el minuto estamos dejando los test como "nice to have" por temas de tiempo de desarrollo 😬